src/components/layout/AppBar.vue

```diff
@@ -231,6 +255,18 @@ export default class AppBar extends Mixins(StateMixin, ServicesMixin) {
     return this.topNavPowerDevice?.status === 'on'
   }
 
+  get topNavPinLight (): string {
+    return this.$store.state.config.uiSettings.general.topNavPinLight
+  }
+
+
```

Comment:
You will need to remove this extra empty line, I can't push to your `develop` branch from outside GitHub.